git-branchless/tests/test_navigation.rs

```diff
@@ -909,6 +909,32 @@ fn test_navigation_switch_revset() -> eyre::Result<()> {
         "###);
     }
 
+    {
+        // switching back to "last checkout"
+        let (stdout, _stderr) = git.branchless("switch", &["@{-1}"])?;
+        insta::assert_snapshot!(stdout, @r###"
+        branchless: running command: <git-executable> checkout @{-1}
+        O f777ecc (master) create initial.txt
+        |\
+        | o 62fc20d create test1.txt
+        |
+        @ fe65c1f create test2.txt
+        "###);
+    }
+
+    {
+        // switching back to "last checkout"
+        let (stdout, _stderr) = git.branchless("switch", &["-"])?;
```

Comment:
suggestion: Can you add a test to ensure that `git switch -` is also capable of checking out the previous branch, not just the previous commit? (I believe that's how it works in Git.)

Reply:
Oh, good catch. Done.